nexus/src/external_api/http_entrypoints.rs

```diff
@@ -352,10 +352,33 @@ pub(crate) fn external_api() -> NexusApiDescription {
         api.register(device_auth::device_auth_confirm)?;
         api.register(device_auth::device_access_token)?;
 
-        api.register(probe_list)?;
-        api.register(probe_view)?;
-        api.register(probe_create)?;
-        api.register(probe_delete)?;
+        Ok(())
+    }
+
+    fn register_experimental<T>(
+        api: &mut NexusApiDescription,
+        endpoint: T,
+    ) -> Result<(), String>
+    where
+        T: Into<ApiEndpoint<Arc<ServerContext>>>,
+    {
+        let mut ep: ApiEndpoint<Arc<ServerContext>> = endpoint.into();
+        // only one tag is allowed
+        ep.tags = vec![String::from("hidden")];
```

Comment:
Does this need to fail on the presence of any existing tags?

Reply:
the posture i was going for here was to be able to move endpoint definitions unmodified from experimental -> normal

e.g., the `#[endpoint(... stuff ...)]`  does not have to change

so this is an overwrite rather than a check on pre-existing conditions type of thing